src/test/java/roomescape/MissionStepTest.java

```diff
@@ -2,70 +2,52 @@
 
 import io.restassured.RestAssured;
 import io.restassured.http.ContentType;
+import java.sql.Connection;
+import java.sql.SQLException;
 import java.util.HashMap;
+import java.util.List;
 import java.util.Map;
 import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.jdbc.core.JdbcTemplate;
 import org.springframework.test.annotation.DirtiesContext;
+import roomescape.domain.reservation.domain.Reservation;
+
+import static org.assertj.core.api.Assertions.assertThat;
 import static org.hamcrest.CoreMatchers.*;
```

Comment:
Google Java Style Convention 중

> 3.3 Import statements
3.3.1 No wildcard imports
Wildcard imports, static or otherwise, are not used.
3.3.3 Ordering and spacing
Imports are ordered as follows:

> - All static imports in a single block.
> - All non-static imports in a single block.

> If there are both static and non-static imports, a single blank line separates the two blocks. There are no other blank lines between import statements.